### PR TITLE
fix: edited heading and subheadings

### PR DESCRIPTION
**Pull request by Sumansourabh14**

### CODE REVIEW

**Comment by Ashish-khanagwal** [comment]
Hey, Suman Saurabh. Thank you, for contributing. Please make sure all the links in the index are directing to their content

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   * [CSS](#css)
   * [JavaScript](#javascript)
   * [React](#react)
-- [WEB-3](#web-3)
+- [Web3](#web-3)
```
Comment (Ashish-khanagwal) [inline README.md:52]: make sure to change it to `[Web3](#web3)` so, that it can direct to the content listed below

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sumansourabh14) [reply README.md]: Thanks for the review, corrected.

**Review by Ashish-khanagwal — APPROVED** [review]
Looking good to me🔥
Make sure to share your contribution on your socials, so that more folks can start contributing to Open Source

**Pull request merged**